### PR TITLE
Completed T11

### PR DESCRIPTION
**Pull request by Ajay07Metamation**

### CODE REVIEW

File: Training/Program.cs [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿//Digital root
+Console.Write ("Enter a Number : ");
+int.TryParse (Console.ReadLine (), out int num);
+while ((num.ToString ().Length) != 1) {
```
Comment (rajeevramanTMM) [inline Training/Program.cs:4]: Store the tostring format in a variable and use it. Remove unnecessary brackets from while loop statement.

File: Training/Program.cs [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+   var elem = num.ToString ().ToCharArray ();
+   num = 0;
+   foreach (char ch in elem) {
```
Comment (rajeevramanTMM) [inline Training/Program.cs:7]: Remove brace for foreach loop only one statement.

File: Training/Program.cs [hunk 3]
```diff
@@ -1,7 +1,12 @@
...
+      num += int.Parse (ch.ToString ());
+   }
+   Array.Clear (elem, 0, elem.Length);
```
Comment (rajeevramanTMM) [inline Training/Program.cs:10]: Do we need this array clearing I worked for me without this line ?

**Review by rajeevramanTMM — CHANGES_REQUESTED** [review]

File: Training/Program.cs (hunk 3 above)
Comment (Ajay07Metamation) [reply Training/Program.cs]: All changes done in latest commit

File: Training/Program.cs [hunk 4]
```diff
@@ -1,7 +1,10 @@
...
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿//Digital root
+Console.Write ("Enter a Number : ");
+int.TryParse (Console.ReadLine (), out int num);
```
Comment (rajeevramanTMM) [inline Training/Program.cs:3]: Fix this.
image

**Review by rajeevramanTMM — CHANGES_REQUESTED** [review]

File: Training/Program.cs (hunk 4 above)
Comment (Ajay07Metamation) [reply Training/Program.cs]: All changes and formatting done in T11

**Pull request closed without merging**